lib/http_clients/creditas/loan_api/loan.ex

```diff
@@ -0,0 +1,47 @@
+defmodule CreditasAcl.LoanApi.Loan do
+  @moduledoc false
+
+  alias HttpClients.Creditas.LoanApi.{
+    Collateral,
+    Contract,
+    Fee,
+    Insurance,
+    InterestRate,
+    Key,
+    Participant,
+    Tax
+  }
+
+  @type t :: %__MODULE__{
+          key: Key.t(),
+          contract: Contract.t(),
+          product: Product.t(),
+          indexation: Indexation.t(),
+          creditor: String.t(),
+          originator: String.t(),
+          underwriter: String.t(),
+          currency: String.t(),
+          financedAmount: Decimal.t(),
+          installmentsCount: integer(),
+          installmentFrequency: String.t(),
+          installmentFixedAmount: Decimal.t(),
+          firstInstallmentDueDate: Date.t(),
+          lastInstallmentDueDate: Date.t(),
+          amortizationMethod: String.t(),
+          collaterals: List.t(Collateral.t()),
+          participants: List.t(Participant.t()),
+          fees: List.t(Fee.t()),
+          taxes: List.t(Tax.t()),
+          interestRates: List.t(InterestRate.t()),
+          insurances: List.t(Insurance.t())
+        }
+
+  @derive Jason.Encoder
+  @enforce_keys ~w(key creditor originator underwriter product currency financedAmount installmentsCount
+    installmentFrequency firstInstallmentDueDate lastInstallmentDueDate interestRates amortizationMethod
+    indexation contract collaterals participants)a
```

Comment:
```suggestion
  @enforce_keys ~w(key creditor originator underwriter product currency financedAmount installmentsCount
                                  installmentFrequency firstInstallmentDueDate lastInstallmentDueDate interestRates amortizationMethod
                                  indexation contract collaterals participants)a
```
:nail_care: pra ficar igual aos outros projetos

Reply:
apliquei aqui, mas n vejo vantagem em adicionar esses tabs a mais

Reply:
é o padrão que a gente adotou em todos os serviços, dá pra mudar se todo mundo concordar :)
@bcredi/developers